templates/themes/maison/layout/footer.html.twig

```diff
@@ -18,7 +18,7 @@
                                 <a href="#" class="nav-link">GET HELP</a>
                             </li>
                             <li class="nav-item">
-                                <a href="#" class="nav-link">CONNECT WITH US</a>
+                                <a href="/connect-with-us" class="nav-link">CONNECT WITH US</a>
```

Comment:
future: we can introduce new twig helper for this part - content_by_remote_id('connect-with-us') or simply remote(...) and set this remote_id in migration